include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -960,7 +960,13 @@ class CommonOperatorLoader : public ProtobufLoader {
       RETURN_IF_ERR(loadIdentity(op, dict));
       return true;
     }
-    if (typeName == "Identity") {
+    if (typeName == "EnforceFinite") {
+      // Load as Identity operation, because in Caffe2 this op does not change
+      // any data. It only raises error if some tensor element is Inf or NaN.
```

Comment:
It doesn't seem correct to emit an Identity op for EnforceFinite, since it drops the "enforce finite" part of the semantics.  I'd rather us support this op correctly or not at all, unless you have a compelling reason for doing it this way.

Reply:
OK. Is there a preferred way to raise an error from Backend layer? We've never done this before. We do have `assert`s, but they are only used in Debug for test purposes.

Reply:
I'm not entirely sure we can...  I mean, obviously we could do something in the interpreter or cpu backend, but ASICs don't even necessarily provide control flow primitives that would allow us to implement it...

Reply:
In the past, we supported ops omitting some functionality. For example, there were `EnsureCPUOutput`, `CopyCPUToMKL`, `CopyMKLToCPU` and `Copy` found somewhere in the nets we wanted to run. Not to change the nets themselves, we added these ops as identity. Alternative would be to say that these actions don't make sense in Glow. Therefore won't be supported. And higher level framework (Caffe2) should take care of removing these ops.

But at that time we didn't have any integration, and goal was to load as much of the protobuf as possible (managed to load whole model as result).

Reply:
I created PR #2340 to add Alias op separately. I am going to close this PR, since not supporting `EnforceFinite` at all is probably the right thing to do.